src/app/test-record/vehicle/vehicle.component.html

```diff
@@ -0,0 +1,66 @@
+<dl *ngIf="testRecord" class="govuk-summary-list">
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">
+      Vehicle identification number (VIN)
+    </dt>
+    <dd class="govuk-summary-list__value" id="test-vin">
+      {{ testRecord.vin | DefaultNullOrEmpty }}
+    </dd>
+  </div>
+  <div *ngIf="testRecord.vehicleType !== 'trl'" class="govuk-summary-list__row">
```

Comment:
This is one for later but we will need some mapping exercise for that like we did for tech-record

(FYI @alleypa @m-ujaffer @rspasici )